PROCESSES.md

```diff
@@ -92,6 +92,13 @@ nothing else, for clarity. Include both the `apis.json` change and
 the project file changes that occur when the project generator has
 been run, in the same commit.
 
+As a final very rare case, a version number such as
+"1.99.99-alpha00" is used to indicate that a breaking change has
```

Comment:
X.99.99-alpha00??

Reply:
I think it's easier to keep it concrete - the "such as" makes it clear that it's an example, I reckon. Let's see what Chris thinks.